docs/examples/plot_lcppn_explainer.py

```diff
@@ -25,23 +25,26 @@
 # Train local classifier per parent node
 classifier.fit(X_train, Y_train)
 
+# Get predictions
+predictions = classifier.predict(X_test)
+
 # Define Explainer
 explainer = Explainer(classifier, data=X_train.values, mode="tree")
 explanations = explainer.explain(X_test.values)
 print(explanations)
 
-# Filter samples which only predicted "Respiratory" at first level
-respiratory_idx = classifier.predict(X_test)[:, 0] == "Respiratory"
-
-# Specify additional filters to obtain only level 0
-shap_filter = {"level": 0, "class": "Respiratory", "sample": respiratory_idx}
+# Filter samples which only predicted "Respiratory"
+respiratory_idx = explainer.get_sample_indices(predictions, "Respiratory")
```

Comment:
What do you think about making it more pandas-like? For example `respiratory_idx = predictions == "Respiratory"`?

Reply:
do you mean to do it roght in plot_lcppn_explainer, or to modify the method?